core/src/main/java/io/parsingdata/metal/expression/value/reference/Count.java

```diff
@@ -29,15 +29,15 @@
 
 public class Count implements ValueExpression {
 
-    public final ValueExpression op;
+    public final ValueExpression operand;
 
-    public Count(final ValueExpression op) {
-        this.op = checkNotNull(op, "op");
+    public Count(final ValueExpression operand) {
+        this.operand = checkNotNull(operand, "operand");
     }
 
     @Override
     public OptionalValueList eval(final Environment env, final Encoding enc) {
-        final OptionalValueList ovl = this.op.eval(env, enc);
+        final OptionalValueList ovl = this.operand.eval(env, enc);
```

Comment:
In some cases I see unnecessary `this.` prefixes. In this case it was already there, and it can be safely removed.

To compare, `toString()` below doesn't use the prefix.